icu_benchmarks/models/wrappers.py

```
@@ -335,17 +402,54 @@ def step_fn(self, element, step_prefix=""):
             loss = self.loss(prediction, target.long(), weight=self.loss_weights.to(self.device)) + aux_loss
-            # Returns torch.long because negative log likelihood loss
+            # Use calibrated probabilities for output transform in test step
+            if step_prefix == "test" and hasattr(self, 'calibrated_model') and self.calibrated_model is not None:
+                transformed_output = self.output_transform((calibrated_tensor, target))
+            else:
+                transformed_output = self.output_transform((prediction, target))
         elif self.run_mode == RunMode.regression:
             # Regression task
             loss = self.loss(prediction[:, 0], target.float()) + aux_loss
         else:
             raise ValueError(f"Run mode {self.run_mode} not yet supported. Please implement it.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)